python/sbp/client/loggers/rotating_logger.py

```diff
@@ -77,18 +42,18 @@ class RotatingFileLogger(JSONLogger):
   """
 
   def __init__(self, filename, when='M', interval=30, backupCount=3,
-               tags={}, dispatcher=dispatch):
+               tags={}, dispatcher=None):
```

Comment:
Why is dispatcher here now defaulting to `None`? I think this needs to be defined for the parent class, which will swallow an exception and only log undispatched SBP because the message dispatch table is `None`.